src/mds/MDSRank.h

```diff
@@ -583,8 +586,9 @@ class MDSRank {
     MDSContext::que replay_queue;
     bool replaying_requests_done = false;
 
-    std::map<mds_rank_t, MDSContext::vec > waiting_for_active_peer;
-    std::map<epoch_t, MDSContext::vec > waiting_for_mdsmap;
+    std::map<mds_rank_t, MDSContext::vec> waiting_for_active_peer;
+    std::map<mds_rank_t, MDSContext::vec> waiting_for_bootstrapping_peer;
```

Comment:
nit: waiting_for_bootstrapped_peer?

Reply:
I actually deliberately chose "bootstrapping" because that's what the peer is doing. It could go either way though...

Reply:
ack